website/manual.md

```diff
@@ -312,57 +312,51 @@ file_server --reload
 
 ### Run subprocess
 
-```
+[API Reference](https://deno.land/typedoc/index.html#run)
```

Comment:
I noticed the run jsdocs are bare... While you're doing this, would you mind adding an example and some text to run's jsdoc?

Reply:
Yes I noticed that too, actually I planned on opening an epic issue to fill out missing docs because most of API is still undocumented

Reply:
Please take a stab at it in this PR. It doesn't need to be complete - but i think it will help get the ball rolling.